src/index.ts

```diff
@@ -0,0 +1,79 @@
+import express from "express";
+import fs from "fs";
+import { nextTick } from "process";
+import { checkNote } from "./api/checkNote";
+
+import { notesApi } from "./api/notesApi";
+import { Note } from "./config/note";
+import { dateFormatter } from "./utils/helper";
+import { validate } from "./utils/validation";
+
+const app = express();
+const jsonParser = express.json();
+const port = process.env.PORT || 3000;
+app.use(function (request, response, next) {
+  let data = `${dateFormatter()} ${request.method} ${request.url} ${request.get(
+    "user-agent"
+  )}`;
+  console.log(data);
+  fs.appendFile("server.log", data + "\n", function () {});
+  next();
+});
+app.get("/", (request, response) => {
+  response.send(`<h1>Hello </h1>`);
+});
```

Comment:
Let's try to use middleware methodology here, and move all the routes to the constants:
`app.get(ROUTES.HOME, asyncMiddleware(defaultHandler))`

Reply:
asyncMiddleware its an util function (place into appropriate folder please)
```
export default (fn) => (req, res, next) => {
  Promise.resolve(fn(req, res, next)).catch(next);
};
```